crates/builder/tests/playground.rs

```diff
@@ -7,17 +7,24 @@ use fs_err as fs;
 #[test]
 fn test_playground() {
     let dir = Utf8PathBuf::from("../../examples/playground");
-    let gen = dir.join("gen");
+    let gen = dir.join("assets").join("gen");
 
     cargo(&dir, ["clean"]);
-    fs::remove_dir_all(&gen).unwrap();
+    cargo(
+        &dir,
+        ["build", "-p=wasm", "--target=wasm32-unknown-unknown"],
+    );
+
+    if gen.exists() {
+        fs::remove_dir_all(&gen).unwrap();
+    }
 
     cargo(&dir, ["build"]);
 
-    let out = dir.join("target").join("playground");
+    let out = dir.join("target").join("assets");
 
     insta::assert_snapshot!(out.ls_ascii(0).unwrap(), @r###"
```

Comment:
Move this to after the release build to verify that as well. And add an assert_snapshot for the wasm package as well. The `.wasm` file checksums can be redacted (i.e. replaced with something like `<redacted>`).